packages/cli/src/networks/index.ts

```diff
@@ -107,8 +106,11 @@ export async function fetchBootnodes(network: NetworkName): Promise<string[]> {
   if (bootnodesFileUrl === null) {
     return [];
   }
-
-  const bootnodesFile = await got.get(bootnodesFileUrl).text();
+  const res = await fetch(bootnodesFileUrl);
+  if (!res.ok) {
+    throw new Error(`Failed to fetch bootnodes file. Status: ${res.status} ${res.statusText}`);
```

Comment:
Based on how `fetchBootnodes` function is called we would duplicate this log

see actual place where error is called
https://github.com/ChainSafe/lodestar/blob/6675739aba0aacd904109bf06faaf74a7b41ec6d/packages/cli/src/networks/index.ts#L125

This is an example on how the complete log looked like previously (from https://github.com/ChainSafe/lodestar/pull/6279)

```
Error fetching latest bootnodes: HTTPError: Response code 404 (Not Found)
```

Reply:
Sure, alighed it.